tests/test_extending_bakery.py

```diff
@@ -15,51 +15,52 @@ def gen_age():
     return 18
 
 
-class KidMommy(mommy.Mommy):
-    age_list = range(4, 12)
+class ExperientBaker(baker.Baker):
+    age_list = range(40, 60)
     attr_mapping = {'age': gen_from_list(age_list)}
 
 
-class TeenagerMommy(mommy.Mommy):
+class TeenagerBaker(baker.Baker):
     attr_mapping = {'age': gen_age}
 
 
-class SadPeopleMommy(mommy.Mommy):
+class SadPeopleBaker(baker.Baker):
     attr_mapping = {'happy': gen_opposite, 'unhappy': gen_opposite}
 
 
 @pytest.mark.django_db
-class TestSimpleExtendMommy:
+class TestSimpleExtendBaker:
     def test_list_generator_respects_values_from_list(self):
-        mom = KidMommy(Person)
-        kid = mom.make()
-        assert kid.age in KidMommy.age_list
+        experient_baker = ExperientBaker(Person)
+        experient_baker = baker_.make()
+        assert experient_baker.age in ExperientBaker.age_list
+        assert kid.age in ExperientBaker.age_list
 
 
 @pytest.mark.django_db
-class TestLessSimpleExtendMommy:
+class TestLessSimpleExtendBaker:
     def test_nonexistent_required_field(self):
         gen_opposite.required = ['house']
-        mom = SadPeopleMommy(Person)
+        baker_ = SadPeopleBaker(Person)
```

Comment:
```suggestion
        sad_baker = SadPeopleBaker(Person)
```
Since you accepted my previous suggestions, I think that all `baker_` variables should be renamed (sorry for not making this explicit before).

Reply:
I've changed it to make more explicit what this test is doing. It's overriding the base class `Baker`. So I decided to leave it more explicit by renaming it to `sad_people_factory`, aka Bolsonaro's government.

Reply:
hahaha :(